NuCivic/visualization_entity
# Add help text to chart form fields with bootstrap popover CIVIC-6095

CIVIC-6095
CIVIC-6083
CIVIC-6084
CIVIC-6089
CIVIC-6092

## Description
Creating a chart is hard, what do all these fields do? We need tooltips!!

## QA Steps
Create or edit a chart, check the help displayed when clicking the (?) icon on the following fields:
### Step Two
- [ ] Series 
- [ ] Y-Field data type
- [ ] X-Field
- [ ] X-Field data type
### Step Three
- [ ] Hovering over the chart types should display the chart type name
### Step Four
- [ ] Query Editor 
- [ ] Filter Editor
- [ ] ~~Transition Time~~ (removed field)
- [ ] Goal
- [ ] Sort
- [ ] Margin
- [ ] Chart height
- [ ] Show title
- [ ] Show controls
- [ ] Show legend
- [ ] Group by X-Field
- [ ] Show Tooltips
- [ ] Reduce Ticks
**X Axis**
- [ ] X format
- [ ] X Label rotation
- [ ] X Axis Label
- [ ] X Axis Label Distance
- [ ] Tick/step Values
**Y Axis**
- [ ] Y format
- [ ] Y Axis Label
- [ ] Y Axis Label Distance
- [ ] Tick/step Values
**linePlusBarChart**
- [ ] Y-1 format
- [ ] Y-1 Axis Label
- [ ] Y-1 Axis Label Distance
- [ ] Y-1 Tick/step Values
- [ ] Y-2 format
- [ ] Y-2 Axis Label
- [ ] Y-2 Axis Label Distance
- [ ] Y-2 Tick/step Values
**Pie Chart**
- [ ] Donut

## Required PR
- [x] https://github.com/NuCivic/dkan/pull/1856 (adds `fa-question-circle-o` icon)
- [x] https://github.com/NuCivic/recline.view.nvd3.js/pull/42
